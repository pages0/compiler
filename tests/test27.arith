let maxPair: ( int * int ) -> int =
    fun (p: (int * int)) : int =
    	if fst p > snd p then
	fst p
	else
	snd p
in maxPair ( 2 , 7)
	