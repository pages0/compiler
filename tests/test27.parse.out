( let [ maxPair ( fun ( p ) ( if ( > ( fst p ) ( snd p ) ) ( fst p ) ( snd p ) ) ) ] ( maxPair ( tuple 2 7 ) ) )